src/main/java/Command.java

```diff
@@ -0,0 +1,8 @@
+public abstract class Command {
+    public String run(TodoList tasks, Storage storage) {
+        storage.save(tasks);
+        return null;
+    }
+
+    public abstract boolean isExit();
```

Comment:
Something that can be improved and eventually retired since it only returns true for one child class. I understand that it is due to the need to break out of the while loop in the Duke run() method, but perhaps a better way is to call System.exit(0) in your run() method in ExitCommand class.